example/javalib/linting/6-pmd/build.mill

```diff
@@ -0,0 +1,35 @@
+// `PmdModule` Performs checks on Java source files using https://pmd.github.io/[PMD] and generates reports from these checks.
```

Comment:
This example needs to be included from the documentation. Like we did here: https://github.com/com-lihaoyi/mill/blob/aa09978646b6b5c02ccd133bc6d01b46d48bbf08/website/docs/modules/ROOT/pages/javalib/linting.adoc?plain=1#L17

Reply:
This file should also contain a `/** Usage */` block containing the executable test cases demoing how it is supposed to be used.